mail_composer_cc_bcc_account/wizards/account_move_send.py

```diff
@@ -0,0 +1,139 @@
+# Copyright 2024 Camptocamp
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+
+from odoo import Command, api, fields, models, tools
+
+
+class AccountMoveSend(models.TransientModel):
+    _inherit = "account.move.send"
+
+    partner_cc_ids = fields.Many2many(
+        "res.partner",
+        "account_move_send_res_partner_cc_rel",
+        "wizard_id",
+        "partner_id",
+        string="Cc",
+        compute="_compute_mail_partner_cc_bcc_ids",
+        store=True,
+        readonly=False,
+    )
+    partner_bcc_ids = fields.Many2many(
+        "res.partner",
+        "account_move_send_res_partner_bcc_rel",
+        "wizard_id",
+        "partner_id",
+        string="Bcc",
+        compute="_compute_mail_partner_cc_bcc_ids",
+        store=True,
+        readonly=False,
+    )
+
+    def _get_default_mail_partner_cc_ids(self, move, mail_template):
+        partners = self.env["res.partner"].with_company(move.company_id)
+        if mail_template.email_cc:
+            for mail_data in tools.email_split(mail_template.email_cc):
+                partners |= partners.find_or_create(mail_data)
+        return partners
+
+    def _get_default_mail_partner_bcc_ids(self, move, mail_template):
+        partners = self.env["res.partner"].with_company(move.company_id)
+        if mail_template.email_bcc:
+            for mail_data in tools.email_split(mail_template.email_bcc):
+                partners |= partners.find_or_create(mail_data)
+        return partners
+
+    @api.model
+    def default_get(self, fields_list):
+        company = self.env.company
+        res = super().default_get(fields_list)
+        partner_cc = company.default_partner_cc_ids
+        if partner_cc:
+            res["partner_cc_ids"] = [Command.set(partner_cc.ids)]
+        partner_bcc = company.default_partner_bcc_ids
+        if partner_bcc:
+            res["partner_bcc_ids"] = [Command.set(partner_bcc.ids)]
+        return res
+
+    @api.depends("mail_template_id")
+    def _compute_mail_partner_cc_bcc_ids(self):
+        for wizard in self:
+            if wizard.mode == "invoice_single" and wizard.mail_template_id:
+                wizard.partner_cc_ids = self._get_default_mail_partner_cc_ids(
+                    self.move_ids, wizard.mail_template_id
+                )
+                wizard.partner_bcc_ids = self._get_default_mail_partner_bcc_ids(
+                    self.move_ids, wizard.mail_template_id
```

Comment:
```suggestion
                    wizard.move_ids, wizard.mail_template_id
                )
                wizard.partner_bcc_ids = self._get_default_mail_partner_bcc_ids(
                    wizard.move_ids, wizard.mail_template_id
```
you are iterating over possibly multiple wizards